tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.ts

```diff
@@ -31,7 +32,12 @@ const DAY_IN_MS = 24 * 1000 * 60 * 60;
 @Component({
   selector: 'line-chart-axis',
   template: `
-    <div [class]="axis + '-axis'">
+    <div
+      [class]="axis + '-axis axis'"
+      [matMenuTriggerFor]="manualControl"
```

Comment:
Having the axis-es themselves be click targets is certainly convenient.  However, this would directly conflict with a future plan to make the x-axis clickable for a "linked time" mode.  In that separate proposal, the x-axis may be clickable to set a chart's "current step" to a specific value, or to a range if click+drag.  I think it's also reasonable that in the future, TB might adopt gestures from line charts in other tools such as showing a vertical/horizontal line when hovering over the x/y axis.

WDYT about starting out without the "axis as click target", and instead starting with these menus triggered from a new option inside 3-dot menu on each scalar card?

Reply:
> However, this would directly conflict with a future plan to make the x-axis clickable for a "linked time" mode. In that separate proposal, the x-axis may be clickable to set a chart's "current step" to a specific value, or to a range if click+drag

I think we could fix this when we get there. Overflow menu is very cumbersome and imo, counter-intuitive in current manifestation of the feature..